.github/workflows/android.yml

```diff
@@ -109,3 +112,17 @@ jobs:
           path : |
             ./**/build/reports/tests/**
             ./**/build/reports/androidTests/connected/**
+
+      - name: Upload unit test coverage to Codecov
+        uses: codecov/codecov-action@v3
+        with:
+          files: build/reports/jacoco/generateUnitTestCoverageReport/generateUnitTestCoverageReport.xml
+          fail_ci_if_error: false
+          flags: unittests-android, unittests
```

Comment:
Note that we don't seem to need the Codecov token as this is a public repo using Github Actions. Can/should we remove the token for iOS/Rust? @langleyd

Reply:
I was getting some 404s sporadically that match [this issue](https://github.com/codecov/codecov-action/issues/557#issuecomment-1216749652), which form what I gathered is apparently due to codecov hitting their github rate limit in cases where the codecov token is not provided for public repos.

Anecdotally for me(as some others in that issue) adding the token resolved it.

Reply:
[`ef87e69`](<https://github.com/matrix-org/matrix-rich-text-editor/commit/ef87e693f3663a12481b0981cc4149fe056d418c>)